### DISCUSSION TITLE
Not Working on Mabook Air M2 OS Sonoma

### DISCUSSION THREAD

**Discussion by reded247** [Ascoware/get-iplayer-automator#487] [General]
Hi all, I've just upgraded to a MacBook Air M2 running OS Sonoma. 

I’m having issues with Get iPlayer Automator 1.26.1 
I just keeping getting the progress bar “Updating Program Index Feeds” running back and forth at the bottom of the page. 

I can add shows through the URL, but it won’t let me download them. 

My log just shows this: 
Get iPlayer Automator 1.26.1 Initialized.
Updating Programme Index Feeds...

Retrieving Metadata For PID 

Any idea on how to fix?

**Comment by skovatch** [comment]
Since you are on an M2, my guess is that Rosetta is either missing or not enabled. First, quit GiA if it's running, then, in the Finder select Get iPlayer Automator.app and then choose Get Info. Click the 'Open in Rosetta' checkbox.

Let me know if that fixes it.

**Reply by reded247** [reply]
Perfect, that sorted the problem straight away.

Thanks so much
